modules/services/syncthing.nix

```diff
@@ -9,10 +9,20 @@ with lib;
     services.syncthing = {
       enable = mkEnableOption "Syncthing continuous file synchronization";
 
-      tray = mkOption {
```

Comment:
This change is not backwards compatible. You will need to do some form of aliasing, e.g., using `mkRenamedOptionModule`.

Reply:
@rycee I'm not really sure how to use `mkRenamedOptionModule` in this context, because `services.syncthing.tray` will continue to be used (now becoming a set instead of a bool). If I try to, I get an infinite recursion. I can hack it by conditioning on `builtins.typeOf`, but I'm wondering if you know of a cleaner way

Reply:
Yeah, I'm thinking that it might be necessary to leave the `tray` option as is and add an option along the lines of
```
trayCommand = mkOption {
  type = types.str;
  default = "${pkgs.qsyncthingtray}/bin/QSyncthingTray";
  defaultText = literalExample ''"''${pkgs.qsyncthingtray}/bin/QSyncthingTray"'';
  description = "…";
};
```
next to it.